app_integrations/apps/app_base.py

```diff
@@ -167,12 +170,16 @@ def _sleep(self):
     def _initialize(self):
         """Method for performing any startup steps, like setting state to running"""
         # Perform another safety check to make sure this is not being invoked already
-        if self._config.is_running:
+        # If this is a successive invocation to a previous execution, allow it to continue
```

Comment:
This comment is slightly confusing, should it just be `This prevents multiple Lambda functions from running simultaneously`?